ethereum/contracts/relayer/wormholeRelayer/WormholeRelayerSend.sol

```diff
@@ -195,7 +225,37 @@ abstract contract WormholeRelayerSend is WormholeRelayerBase, IWormholeRelayerSe
                 refundChain,
                 refundAddress,
                 deliveryProviderAddress,
-                vaaKeys,
+                WormholeRelayerSerde.vaaKeyArrayToMessageKeyArray(vaaKeys), // why doesn't this work without the 'WormholeRelayerSerde.' ?
```

Comment:
I think you'd have to write vaaKeys.vaaKeyArrayToMessageKeyArray() if you wanted to do this. Not certain, but worth a shot